### DISCUSSION TITLE
Can I check how many remote viewers are currently connected?

### DISCUSSION THREAD

**Discussion by Leiches** [cables-gl/cables_docs#701] [Answers & Questions]
Hi - Remote Viewer is a beta feature and i don´t know how much you can do with it, but is there a way to check how many useres are connected to my patch?
Or is there even a way how you can display every viewer on your patch and show other users what everyone is doing?

**Comment by steam0r** [comment]
not sure i understand. you want the data that we display in the multiplayer bar in ops? this would then be bound
to the cables editor and not work in export...or am i not getting your usecase?
